### DISCUSSION TITLE
Does it work for every command?

### DISCUSSION THREAD

**Discussion by WhoMI7** [CogentRedTester/mpv-scripts#20] [Q&A] [answered]
I can use
 ```
x                set ab-loop-a ${=time-pos}
c                set ab-loop-b ${=time-pos}
 \                set ab-loop-a no ; set ab-loop-b no ; show-text "Clear A-B loop"
```
individually, but If I want to use something like `l script-message cycle-commands "set ab-loop-a; show-text '${=time-pos}'" "set ab-loop-b; show-text '${=time-pos}'" "set ab-loop-a no; set ab-loop-b no; show-text 'Clear A-B loop'"`
I get this error 
![image](https://user-images.githubusercontent.com/93109835/215458801-03687745-c07a-4f4e-81e3-0252dbc5f80e.png)

Logs: [mpv.log](https://github.com/CogentRedTester/mpv-scripts/files/10535129/mpv.log)

**Comment by CogentRedTester** [comment, marked as answer]
It doesn't work because you're not using the same commands. You're doing `set ab-loop-a` but not setting it to anything, instead you'd doing the show-text command. Just use the commands that you listed above.
